__tests__/app.test.js

```diff
@@ -33,4 +33,42 @@ describe("GET api/topics", () => {
                 });
         });
     });
-});
\ No newline at end of file
+});
+
+describe("GET api/articles", () => {
+    describe("Connecting to path", () => {
+        test("Should return status 200", () => {
+            return request(app).get("/api/articles").expect(200)
+        });
+        test("Should return the all the articles with correct properties", () => {
+            return request(app).get("/api/articles").expect(200)
+                .then(({ body: {articles} }) => {
+                    expect(articles.length).toBe(13)
+                    articles.forEach((article) => {
+                        expect(article).toHaveProperty("author", expect.any(String))
+                        expect(article).toHaveProperty("title", expect.any(String))
+                        expect(article).toHaveProperty("article_id", expect.any(Number))
+                        expect(article).toHaveProperty("topic", expect.any(String))
+                        expect(article).toHaveProperty("created_at", expect.any(String))
+                        expect(article).toHaveProperty("votes", expect.any(Number))
+                        expect(article).toHaveProperty("article_img_url", expect.any(String))
+                        
+                    });
+                });
+        });
+        test("Should include the comment_count", () => {
```

Comment:
You can include comment_count in the test above which would make your code more dry